jekyll/_cci2/arm_resources.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: classic-docs
+title: "ARM resources"
+short-title: "Using ARM resources on CircleCI"
+description: "Using ARM resources on CircleCI"
+version:
+- Cloud
+---
+
+# Overview
+
+This document will walk you through the setup steps required to use an ARM
```

Comment:
We capitalise this as `Arm` in the marketing materials, would it make sense to use that capitalisation in the docs too?

Reply:
Yep, that makes sense.